bigint/src/bigint.rs

```diff
@@ -1105,3 +1189,36 @@ impl BigInt {
         return Some(self.div(v));
     }
 }
+
+/// Perform in-place two's complement of the given digit range,
+/// in little-endian byte order.
+#[inline]
+fn twos_complement_le<T>(digits: &mut [T])
```

Comment:
I think it's harmful to clarity for this to be generic on `T`.  This is internal, we know it's `u8`.

But it might be beneficial to take `I: Iterator<Item=u8>` instead of a slice, and then we only need one copy of the function.  The `be` caller can `rev()` the iterator they pass in.